<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetThresholdDlg</class>
 <widget class="QDialog" name="SetThresholdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>250</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxBox">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::up-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;
	border-top-right-radius:3px;
	image: url(:/new/prefix1/image/up.png);
}

QSpinBox::down-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;	
	border-bottom-right-radius:3px;
	image: url(:/new/prefix1/image/down.png);
}
QSpinBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>110</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最小值：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>250</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::up-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;
	border-top-right-radius:3px;
	image: url(:/new/prefix1/image/up.png);
}

QSpinBox::down-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;	
	border-bottom-right-radius:3px;
	image: url(:/new/prefix1/image/down.png);
}
QSpinBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="titleLab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阈值设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>统一设置</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最大值：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="mindoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox::up-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;
	border-top-right-radius:3px;
	image: url(:/new/prefix1/image/up.png);
}

QDoubleSpinBox::down-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;	
	border-bottom-right-radius:3px;
	image: url(:/new/prefix1/image/down.png);
}
QDoubleSpinBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="maxdoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox::up-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;
	border-top-right-radius:3px;
	image: url(:/new/prefix1/image/up.png);
}

QDoubleSpinBox::down-button
{
	width: 35px;
	height: 25px;
	border-left-width:1px;
	border-left-color:darkgray;
	border-left-style:solid;	
	border-bottom-right-radius:3px;
	image: url(:/new/prefix1/image/down.png);
}
QDoubleSpinBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">setthreshold/mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
